### DISCUSSION TITLE
Google Chrome Bitwarden self-host Login "Failed to fetch"

### DISCUSSION THREAD

**Discussion by Reaper-97** [dani-garcia/vaultwarden#4913] [Q&A]
Hi there,

i setup vaultwarden on my unraid server and connected it via reverse proxy to my cloudflare domain but i cant login to the official browser plugin for chrome even with entered self host ip. Always getting failed to fetch error.

Hoepfully someone can help me. (vaultwarden on unraid / chrome browser / bitwarden plugin are all updated to newest version)

**Comment by REZISTENCE** [comment]
I to am getting an error, using the BitWarden Web plugin in firefox and chrome
![BW Error](https://github.com/user-attachments/assets/81058e82-55b1-4a57-b96f-89e2b0e44832)
I have set it to the self-hosted reverse proxy, which works fine on my android phone with the Bitwarden apk
WEB Plugin settings
![login](https://github.com/user-attachments/assets/dbd1fa81-faa4-40fb-a999-f94b3a26a7ba)
![self](https://github.com/user-attachments/assets/193cc1fd-f936-4b2e-bf5f-ba4d9a186ef3)

I have also upgraded the Vaultwarden docker container and tried older web plugins just in case

Is anyone having this same issue???

**Comment by j4care-christianhahn** [comment]
same issue on my side since update to latest vaultwarden
